File: Code/hello.c
/* problemSet1 */

#include<stdio.h>
#include<math.h>

double main()
{
     int count;
     double z, y, x;
     printf("HELLO FROM C\n");
     for(count = 1; count<11; count++)
     {
          z=count;
          x=1/z;
          y=sin(x);
          printf("%d %lf %lf\n",count,x,y);
     }
return 0;
}

File: Code/Hello.f
c23456789
      program problemSet1Fortran
	 
	 integer i, n
	 real y
	 write(*,*) 'HELLO FROM FORTRAN'
	 do 10 i = 1, 10
          y=i
          write(*,*) i,'', 1/y,'', sin(1/y)
10    continue

      end
